daemon/smartcard.c

```diff
@@ -902,6 +903,9 @@ smartcard_container_change_pin(smartcard_t *smartcard, control_t *control, conta
 	out.pairing_secret.data = mem_memcpy(pair_sec, sizeof(pair_sec));
 
 	ret = protobuf_send_message(smartcard->sock, (ProtobufCMessage *)&out);
+	mem_memset0(out.token_pin, strlen(out.token_pin));
+	mem_memset0(out.token_pin, strlen(out.token_pin));
```

Comment:
the second mem_memset0 should be applied to token_newpin

Reply:
Done